automata.js.org

notscrappie opened the pull request:
Will be used for the documentation page of Automata. You can check the website hosted on Vercel here: https://automata-shadowrunners.vercel.app. Don't know if I set it up correctly so let me know!

- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 commented on cnames_active.js:
```diff
@@ -245,6 +245,7 @@ var cnames_active = {
   "auto-task-doc": "cname.vercel-dns.com", // noCF
   "autodocs": "bguiz.github.io/autodocs", // noCF? (don´t add this in a new PR)
   "automic": "automicjs.github.io",
+  "automata": "cname.vercel-dns.com", // noCF
```
```suggestion
  "automata": "cname.vercel-dns.com", // noCF
  "automic": "automicjs.github.io",
```
As the CI error notes, your entry needs to be in alphabetical order

notscrappie replied:
Edited it, should be okay now.

MattIPv4 reviewed (approved):
✅ Commit valid, site has content based on https://automata-shadowrunners.vercel.app/

I trust that your site is configured ready to work with the requested js.org subdomain.

indus commented:
@notscrappie sorry for the long delay. I've  added your requested subdomain to JS.ORGs zonefile. But Iget redirected to www.automata.js.org which does not exist. You may have to configure your host.
I'll merge after we have "contact". Please keep this PR open.

notscrappie commented:
> @notscrappie sorry for the long delay. I've added your requested subdomain to JS.ORGs zonefile. But Iget redirected to [www.automata.js.org](http://www.automata.js.org) which does not exist. You may have to configure your host. I'll merge after we have "contact". Please keep this PR open.

No worries! I removed the record that was redirecting to the www version of the domain and it now works. Thanks for adding my subdomain!

indus commented:
Welcome https://automata.js.org
